mx-platform-aspnet-core/Program.cs

```diff
@@ -123,3 +135,9 @@ string ConvertToSnakeCase(object camelized)
 });
 
 app.Run();
+
+public class Request
+{
+  public string? user_guid { get; set; }
+  public string? user_id { get; set; }
```

Comment:
I would use the capitalized CamelCase version instead of the snake_case for these properties, it is the C# standard.

Reply:
I was thinking I would have to match the data coming in the request exactly but maybe that's not the case?

Reply:
Fixed with Lu's pull request.